src/interpret.jl

```diff
@@ -16,7 +16,15 @@ end
 function lookup_expr(frame, e::Expr)
     head = e.head
     head == :the_exception && return frame.framedata.last_exception[]
-    head == :static_parameter && return frame.framedata.sparams[e.args[1]::Int]
+    if head == :static_parameter
+        arg = e.args[1]::Int
+        if isassigned(frame.framedata.sparams, arg)
+            return frame.framedata.sparams[arg]
+        else
+            syms = sparam_syms(frame.framecode.scope)
```

Comment:
This is arguably a bit slower than needed but I really don't think this is a case to worry too much about.